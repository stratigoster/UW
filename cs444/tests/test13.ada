PACKAGE foo IS
  type Week is (Sun,Mon,Tue,Wed,Thu,Fri,Sat,Sun,Sun);
--                                          ^^^ ^^^
-- conflicting declarations for Sun
END foo;
