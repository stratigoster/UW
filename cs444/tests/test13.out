***ERROR***:2:54	Conflicting declaration of `sun'
***ERROR***:2:54	Conflicting declaration of `sun'

------
Compilation error
------
